### DISCUSSION TITLE
state.my_variable or GetState('my_variable') ?

### DISCUSSION THREAD

**Discussion by nepomuk16321** [EDCD/EDDI#2661] [EDDI Standalone Q&A]
Could it be that the query of the State() variables via state.my_variable or GetState('my_variable') is different depending on which PC system you are using, Win10 or Win11?
Background to the question: Both variants work for me (Win10), but a CMDR colleague (Win11) often gets errors. We use the context variables. However, our character differs from the original character. It has been translated into German. I have converted all (or almost all) of the queries for the context variables to the GetState variant. My CMDR colleague is now grumbling with me (no kidding), but we want to correct this.

Translated with DeepL.com (free version)

**Comment by Tkael** [comment]
The Windows version should make no difference at all but I have found an issue with state variables that should be resolved in the next beta release (hopefully very soon!)
